File: src/main/java/eu/mshade/enderframe/item/metadata/StoredEnchantmentsItemStackMetadata.java
package eu.mshade.enderframe.item.metadata;

import eu.mshade.enderframe.item.Enchantment;
import eu.mshade.enderframe.metadata.itemstack.ItemStackMetadataKey;
import eu.mshade.enderframe.metadata.itemstack.ItemStackMetadataKeyValue;

import java.util.List;

public class StoredEnchantmentsItemStackMetadata extends ItemStackMetadataKeyValue<List<Enchantment>> {

    public StoredEnchantmentsItemStackMetadata(List<Enchantment> metadataValue) {
        super(ItemStackMetadataKey.STORED_ENCHANTMENTS, metadataValue);
    }
}

File: src/main/java/eu/mshade/enderframe/item/metadata/StoredEnchantmentsItemStackMetadata.kt
package eu.mshade.enderframe.item.metadata

import eu.mshade.enderframe.item.Enchantment
import eu.mshade.enderframe.item.ItemStackMetadataKey
import eu.mshade.enderframe.metadata.MetadataKeyValue

class StoredEnchantmentsItemStackMetadata(metadataValue: List<Enchantment>) :
    MetadataKeyValue<List<Enchantment>>(ItemStackMetadataKey.STORED_ENCHANTMENTS, metadataValue)
